src/libstd/sys/windows/os.rs

```diff
@@ -72,14 +72,18 @@ pub fn error_string(errnum: i32) -> String {
 }
 
 pub struct Env {
+    inner: EnvSlices,
+}
+
+struct EnvSlices {
```

Comment:
To me when I initially read this it wasn't clear why the separation here was needed, could you add a comment explaining why these are separate?